.github/workflows/ml-pipeline-tests.yml

```
@@ -52,102 +52,137 @@ jobs:
+        except (FileNotFoundError, ET.ParseError, TypeError):
+            coverage_str = 'XML not found'
+
+        try:
+            with open('artifacts/metrics.json') as f:
+                metrics = json.load(f)
+            accuracy = metrics.get('accuracy', 0) * 100
+            accuracy_str = f'{accuracy:.2f}%'
+            acc_status = '✅' if accuracy >= 90 else ('⚠️' if accuracy >= 80 else '❌')
+        except (FileNotFoundError, json.JSONDecodeError):
+            accuracy_str = 'JSON not found'
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)